driver.py

```diff
@@ -468,10 +468,12 @@ def generate_driver(p: Panel, options: Options) -> None:
 	module = f"panel-{dash_id}"
 	with open(f'{p.id}/{module}.c', 'w') as f:
 		f.write(f'''\
-// SPDX-License-Identifier: GPL-2.0-only
-// Copyright (c) {datetime.date.today().year} FIXME
-// Generated with linux-mdss-dsi-panel-driver-generator from vendor device tree:
-//   Copyright (c) 2013, The Linux Foundation. All rights reserved. (FIXME)
+/* 
+ * SPDX-License-Identifier: GPL-2.0-only
```

Comment:
The SPDX-License-Identifier should be a `//` comment, see https://www.kernel.org/doc/html/latest/process/license-rules.html#license-identifier-syntax

For the other Copyright text it varies in the kernel tree. Most are using `/* ... */`, but there are also some which are not, see e.g. https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/gpu/drm/panel/panel-samsung-s6e88a0-ams452ef01.c

So I think it might be fine to keep this as-is. What do you think?

Reply:
Ah yes, that SPDX as // struck me around the time i hit the pillow. Well, I'm two phased there, perhaps it's a good idea to encourage people to get to know the script a little and edit as needed as nothing can be guaranteed as perfect, but on the other hand, just grabbing it, generating a panel file and then just editing the output a bit would be optimal as well. 

Cheers and thanks for all the panel drivers!